syntax = "proto3";
package com.panelsw.ca;
option optimize_for = LITE_RUNTIME;

enum ReparsePointAction {
	delete = 0;
	restore = 1;
}

message ReparsePointDetails {
	ReparsePointAction action = 1;
	bytes path = 2;
	bytes reparseData = 3;
}
